qemu/tests/cfg/blockdev_full_backup.cfg

```diff
@@ -0,0 +1,85 @@
+- blockdev_full_backup:
+    virt_test_type = qemu libvirt
+    kill_vm_on_error = yes
+    login_timeout = 240
+    storage_pools = default
+    storage_type_default = "directory"
+    target_path_default = "/var/lib/avocado/data/avocado-vt"
```

Comment:
For ppc test, we might use 'venv', so there is no dir "/var/lib/avocado/data/avocado-vt", could you help to change it to the same dir as the source path?

Reply:
@ngu-niny , please try first.  it's image directory path, I guess it will not impact your test.

Reply:
@xutian In fact, I found the problem when I did the test:

03:13:10 ERROR| virttest.qemu_monitor.QMPCmdError: QMP command 'blockdev-add' failed    (arguments: {'node-name': 'file_dst', 'driver': 'file', 'filename': '/var/lib/avocado/data/avocado-vt/target.qcow2', 'aio': 'threads'},    error message: {'class': 'GenericError', 'desc': "Could not open '/var/lib/avocado/data/avocado-vt/target.qcow2': No such file or directory"})